woocommerce/wc-admin
# [WIP] Try fresh-data wp-plugin as external.

This commit tries out using fresh-data as a wp-plugin that is external
to the repository altogether. It exists here in a subdirectory, but for
a real implementation it should be in its own package.

See #248 and https://github.com/WordPress/gutenberg/pull/8754/files for more details.

To Test:

0. Clone gutenberg to your plugins directory, use current master, `npm install`, `npm run build`.
1. Build this branch as normal.
2. Test out functionality on the orders. It should show "Loading..." and then the orders. And clicking the button should actually fulfill each order.
3. Change order status from another page, the order status on this page should update within 5 minutes.
